CONTRIBUTING.md

```diff
@@ -110,19 +111,20 @@ Some of the SDK configuration options can also be previewed in the demo app by u
 | `useHistory`               | `true`,`false`      | `false` | Enable basic dummy host app history                              |
 | `customisedUI`             | `true`,`false`      | `false` | Enable customised SDK UI                                         |
 | `showUserConsent`          | `true`,`false`      | `false` | Enable user consent step                                         |
+| `customWelcomeScreenCopy`  | `true`,`false`      | `false` | Configure welcome screen with custom text                        |
```

Comment:
Missing documentation for new `multiDocWithBooleanValues` URL query parameter, think my commit for the change must have been after PR was merged.
```suggestion
| `customWelcomeScreenCopy`  | `true`,`false`      | `false` | Configure welcome screen with custom text                        |
| `multiDocWithBooleanValues`   | `true`,`false`      | `false` | Enable document types with boolean flags                     |
```